client-java/BUILD

```diff
@@ -60,4 +60,23 @@ java_library(
         "//dependencies/maven/artifacts/org/slf4j:slf4j-api",
         "//dependencies/maven/artifacts/org/slf4j:slf4j-simple",
     ]
+)
+
+java_test(
+    name = "TransactionTest",
+    srcs = ["src/test/java/ai/grakn/client/TransactionTest.java","src/test/java/ai/grakn/client/ServerRPCMock.java"],
+    deps = [":client-java"] + ["//grakn-graql"] +[
```

Comment:
This is interesting - is it like a style/convention to split the arrays of strings between individual internal dependencies and external dependencies, @marco-scoppetta ?